src/timeline/components/timeline.vue

```diff
@@ -327,6 +331,12 @@ export default {
             this.violationClicked = value.violationClicked;
 		},
 		onViolationClicked(value) {
+            let violationTime = this.timeFormatter.parse(value.violation.violationTime);
+            let end = violationTime + 20000000;
+            let start = violationTime - 20000000;
```

Comment:
Can we define these in a const at the top of the file

Reply:
Changed it to use the timeline padding you had already set instead of using 20000000 value.